pipeline_steps/common.groovy

```diff
@@ -22,10 +22,11 @@ def install_ansible(){
     source .venv/bin/activate
 
     # These pip commands cannot be combined into one.
-    pip install -U six packaging appdirs
-    pip install -U setuptools
     pip install 'pip==9.0.1'
```

Comment:
In OSA we've found it best to update pip and setuptools at the same time:

pip install 'pip==9.0.1 setuptools==33.1.1'

A later version of setuptools causes nonsense.